components/director/internal/open_resource_discovery/fixtures_test.go

```diff
@@ -615,6 +648,41 @@ func fixBundleCreateInput() []*model.BundleCreateInput {
 	}
 }
 
+func fixAPIsWithHash() []*model.APIDefinition {
+	apis := fixAPIs()
```

Comment:
The `Version` value in the returned Apis from `fixAPIs` should be the same as the `Version` value in the `fixORDDocument`

Same for Events/Packages.

Reply:
This breaks a lot of tests on the service layer if you meant to change the version in fixAPIs().
If we put the same version only in the scope of this function, then it is okay